storage-bigtable/src/lib.rs

```diff
@@ -412,6 +413,36 @@ impl LedgerStorage {
         Ok(blocks.into_iter().filter_map(|s| key_to_slot(&s)).collect())
     }
 
+    // Fetches and gets a vector of confirmed blocks via a multirow fetch
+    pub async fn get_confirmed_blocks_with_data(
+        &self,
+        slots: Vec<Slot>,
+    ) -> Result<Vec<(Slot, ConfirmedBlock)>> {
+        debug!(
+            "LedgerStorage::get_confirmed_blocks_with_data request received: {:?}",
+            slots
+        );
+        inc_new_counter_debug!("storage-bigtable-query", 1);
+        let mut bigtable = self.connection.client();
+        let row_keys: Vec<RowKey> = slots.into_iter().map(slot_to_blocks_key).collect();
+        let data: Vec<(Slot, ConfirmedBlock)> = bigtable
+            .get_protobuf_or_bincode_cells::<StoredConfirmedBlock, generated::ConfirmedBlock>(
```

Comment:
if i understand correctly, you're suggesting: 
- get rid of the `collect()` on `row_keys`
- pass through a generic iterator type in `get_protobuf_or_bincode_cells()` and have it return an iterator

i think the benefit would be that you drop a bunch of `collect()` calls, but tradeoff is it makes function signatures of`get_protobuf_or_bincode_cells` harder to read

will make a note and add in a future PR

Reply:
@mvines do you consider this blocking? we have a potential consumer for this method

Reply:
I don't see why this can't be addressed now

Reply:
> I don't see why this can't be addressed now

Thanks!

@0xEdgar is this something you have time to address?  maybe we can pick it up internally if not

Reply:
@t-nelson  just pushed a change, let me know how it looks

Reply:
code credit goes to @buffalu